_store/katjaglass-timekex.md

```diff
@@ -0,0 +1,74 @@
+---
+title: TimeKex - Timetracking - Kimai meets Excel
+name: TimeKex - Timetracking - Kimai meets Excel (Web Application - API utilization)
+intro: "Import your Excel maintained times into Kimai"
+developer: katjaglass
+date: "2022-03-04 12:00:00 +0100"
+icon: fas fa-file-excel
+price: "0"
+screenshot: 
+  - https://raw.githubusercontent.com/KatjaGlassConsulting/TimeKex/main/docs/img/timesheet_to_kimai_01.gif
+  - https://raw.githubusercontent.com/KatjaGlassConsulting/TimeKex/main/docs/img/layout_week_display.png
+  - https://raw.githubusercontent.com/KatjaGlassConsulting/TimeKex/main/docs/img/example_admin_create_02.png
+  - https://raw.githubusercontent.com/KatjaGlassConsulting/TimeKex/main/docs/img/layout_week_display_issues.png
+github: https://github.com/KatjaGlassConsulting/TimeKex
+type: client
+new: true
+features:
+  - excel
+  - import
+tags:
+  - Excel
+  - Import
+---
+
+This WebApplication is designed to allow the tracking of working hours in Excel which can then be uploaded to the Kimai Timetracking application.
+
+![Screenshot of login screen](https://raw.githubusercontent.com/KatjaGlassConsulting/TimeKex/main/docs/img/timesheet_to_kimai_01.gif)
+
+## Documentation and Demo
+
+The documentation of TimeKex as well as a running demonstration instance is accessible [here](https://katjaglassconsulting.github.io/TimeKex/).
+
+[Example excel](https://github.com/KatjaGlassConsulting/TimeKex/tree/main/exampleFiles) files are locted in the GitHub repository.
+
+## Status
+
+TimeKex is available in the current version which does not contain tests. The source code is available under the MIT license which allows a flexible and generic usage and modifications without warrenty.
```

Comment:
I guess you copied over the README.md, correct? 
In my opinion  "which does not contain tests" is not interesting for potential users. And the MIT comes in the next header, so this could be removed entirely. Just saying^^